<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Blue</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/blue_icon.ico</normaloff>:/images/blue_icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>381</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLineEdit" name="usernameTextField">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="passLabel">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string>Mot de passe :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="usernameLabel">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string>Pseudonyme :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="passwordTextField">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>22</height>
        </size>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>321</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0" alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Connexion</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QProgressBar" name="progressBar">
       <property name="maximumSize">
        <size>
         <width>340</width>
         <height>21</height>
        </size>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="textVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="informationLabel">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>15</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
